pkg/analytics/track.go

```diff
@@ -359,9 +359,15 @@ func track(event string, success bool, props map[string]interface{}) {
 		return
 	}
 
-	if !okteto.IsContextInitialized() || (!okteto.Context().Analytics && !okteto.IsOktetoCloud()) {
+	if !okteto.IsContextInitialized() {
+		oktetoLog.Info("failed to send analytics: okteto context not initialized")
```

Comment:
I don't think we should log this. There are many legit reasons foe this no?

Reply:
No, if that happens, is something weird that we should investigate. this is not exposed to the user